Sources/MobileCoinClient+Deprecated.swift

```diff
@@ -209,6 +209,92 @@ extension MobileCoinClient {
         }
     }
 
+    @available(*, deprecated, message:
+        """
+        Use the new `prepareTransaction(...)` that accepts a 32-byte rngSeed.
+
+        ```
+        public func prepareTransaction(
+            to recipient: PublicAddress,
+            memoType: MemoType = .recoverable,
+            amount: Amount,
+            fee: UInt64,
+            rngSeed: RngSeed,
+            completion: @escaping (
+                Result<PendingSinglePayloadTransaction, TransactionPreparationError>
+            ) -> Void
+        ) {
+        ```
+
+        this function creates a 32-byte seed by combining the data from 4 calls to rng.next()
+        """)
+    public func prepareTransaction(
```

Comment:
Is this for backwards compatibility so that Moby can integrate the updated SDK without simultaneously updating to use the new API? 👏

Reply:
ya, it was a public API for a small amount of time so should be deprecated before removal.